Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.16-alpine
+
+ENV config_path $config_path
+RUN mkdir http-server
+WORKDIR http-server
+COPY . .
+
+RUN go mod tidy
+RUN echo $config_path
+RUN go build ./cmd/main.go
```

Comment:
I would recommend to have multi-stage build if you'd like to get rid of source files and use only binary file. Images like `scratch` should help